packages/malloy/src/lang/ast/source-elements/named-source.ts

```diff
@@ -104,9 +104,10 @@ export class NamedSource extends Source {
       return;
     }
     if (entry.type === 'query') {
+      // MTOY TODO techincally we could, if this code ever runs, investigate.
```

Comment:
TODO

Reply:
```
      // I don't understand under what circumstance this code would be
      // executed, what Malloy yo would write to generate this error,
      // but the error exists so I am leaving it for now.
      //
      // Someone with time and courage should either remot this error
      // because it isn't possible, or go ahead and make a source out
      // of a query, which is a thing you can do, although when you
      // do that it currently doesn't go through this path.
 ```